packages/api-client/src/style.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * API Client Styles
+ */
+
+@import "@scalar/themes/style.css"; /* Theme Base Styles and Reset */
+@import "@scalar/components/style.css"; /* Component Library Styles */
+@import "@scalar/themes/tailwind.css"; /* Tailwind Theme + Config */
+
+.scalar-client {
```

Comment:
I tried turning this into `.scalar-app` but we end up with some ordering issues. I'm going to do it when I do the deduping PR.